### DISCUSSION TITLE
Kotlin code for access Android/data folder using Shizuku

### DISCUSSION THREAD

**Discussion by TQ123** [RikkaApps/Shizuku#437] [General]
Hi, I am creating a simple application. When a button is clicked a file should move to Android/data/any package 
Due to Android 13 and 14 Permission restrictions I can't use the SAF method. So I integrated Shizuku SDK into my app. The connection between Shizuku and my app is successful. But I don't know how to copy the file using Shizuku. Anyone, please help me.

FV file manager and MT file manager using this method, I don't know how they are doing that.

i am using these three dependency in my app
    implementation("org.apache.commons:commons-io:1.3.2")
    implementation ("dev.rikka.shizuku:api:13.1.5")
    implementation ("dev.rikka.shizuku:provider:13.1.5")

**Comment by Krishramishra8303636** [comment]
Pkg instoll n

**Reply by ghost** [reply]
adb shell sh /storage/emulated/0/Android/data/moe.shizuku.privileged.api/start.sh
